tests/ios/Podfile

```diff
@@ -1,7 +1,10 @@
 platform :ios, '9.0'
 
+# Allow using RNFirebase as static frameworks
+$RNFirebaseAsStaticFramework = true
+
 # Version override testing
-# $FirebaseSDKVersion = '6.8.1'
+# $FirebaseSDKVersion = '6.15.0'
```

Comment:
Oh come on take it to 6.21.0 :-)
Default to 6.21.0 even... why not?

Reply:
I wish! Will be for v7